data/fields/beauty.json

```diff
@@ -4,9 +4,13 @@
     "label": "Services",
     "strings": {
         "options": {
-            "nails": "Nails",
+            "nails": "Manicure / Pedicure",
             "tanning": "Tanning",
-            "cosmetics": "Cosmetics"
+            "cosmetics": "Cosmetics",
+            "spa": "Hydrotherapy",
```

Comment:
I’m not an expert on this industry, but day spas or medspas (medical spas) are very common. They’re distinct from hydrotherapy facilities, which became virtually nonexistent in the U.S. generations ago, and also completely independent of any natural spas.

[![](https://upload.wikimedia.org/wikipedia/commons/thumb/7/7b/Philipe's_Day_Spa.jpg/640px-Philipe's_Day_Spa.jpg)](https://commons.wikimedia.org/wiki/File:Philipe's_Day_Spa.jpg)

Reply:
thanks, let's use _Day Spa_. I guess the corresponding row of the table on the [wiki](https://wiki.openstreetmap.org/wiki/Key:beauty) could be updated as well. :wink:

Reply:
Updated the wiki, and the list looks good to me now :)